src/unitxt/catalog/cards/llm_as_judge/model_response_assessment/mt_bench.json

```diff
@@ -0,0 +1,9 @@
+{
+    "type": "task_card",
+    "loader": null,
+    "preprocess_steps": [],
+    "task": "tasks.llm_as_judge.rag.model_response_assessment",
```

Comment:
tasks.rag.model_response_assessment

Reply:
Yoav I understand the problem with the fields, but not putting it under "llm_as_judge" will make it so nobody will ever find it. We need to think of a better solution